packages/flame/lib/src/events/messages/drag_cancel_event.dart

```diff
@@ -11,4 +11,6 @@ class DragCancelEvent extends Event {
   final int pointerId;
 
   DragEndEvent toDragEnd() => DragEndEvent(pointerId, DragEndDetails());
+  @override
+  String toString() => 'DragCancelEvent(pointerId:$pointerId)';
```

Comment:
```suggestion
  String toString() => '$DragCancelEvent(pointerId:$pointerId)';
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.

Reply:
Not sure it's worth it, we are not planning to rename these events in the foreseeable future.

Reply:
Agreed. It was just suggestion anyways. 

But it might help in avoiding the classic copy-paste problem too. If a new event class is added and someone just implements `toString` by copying it from an existing event class, they would actually get an import error.

Not too big of a use case though 😅

Reply:
Yeah, it just looks a bit weird -- never seen anyone doing it like that :)

I'm gonna close the suggestions.